core/src/main/java/org/frankframework/pipes/CompressPipe.java

```diff
@@ -240,22 +240,22 @@ public void setFilenamePattern(String string) {
 	}
 
 	/**
-	 * flag indicates whether the message is the content or the path to a file with the contents. for multiple files use ';' as delimiter
+	 * Flag indicates whether the message is the content or the path to a file with the contents. For multiple files, use ';' as a delimiter.
```

Comment:
```suggestion
	 * A flag that indicates whether the message is the content or the path to a file with the contents. For multiple files, use ';' as a delimiter.
```

Keeping it the same as another change made by the AI below...